auth-web/src/views/auth/staff/SetupAccountView.vue

```diff
@@ -2,9 +2,17 @@
   <v-container>
     <div class="view-container">
       <article>
-        <h1 class="mb-3">Create a Director Search Account</h1>
-        <p class="intro-text pb-3">Enter the name and contact information for this account, and indicate which products
-this account will have access to.</p>
+        <div class="d-flex flex-row">
+          <div>
+            <v-btn x-large icon color="secondary" class="mr-2" @click="handleBackButton()" data-test="account-settings-back-button">
+              <v-icon class="font-weight-bold">mdi-arrow-left</v-icon>
+            </v-btn>
+          </div>
+          <div>
+            <h1 class="mb-3">Create a Director Search Account</h1>
+            <p class="intro-text pb-3">Enter the name and contact information for this account, and indicate which products this account will have access to.</p>
```

Comment:
Replace 'intro-text pb-3' selectors with 'pb-0'. The 'view-header' selector it's encapsulate with will dictate spacing.